src/Agent.Plugins/PipelineArtifact/PipelineArtifactPluginV2.cs

```diff
@@ -342,5 +342,11 @@ private async Task<Guid> GetProjectIdAsync(AgentTaskPluginExecutionContext conte
 
             return proj.Id;
         }
+
+        private void OutputBuildInfo(AgentTaskPluginExecutionContext context, int? pipelineId){
+            context.Output(StringUtil.Loc("DownloadingFromBuild", pipelineId));
+            // populate output variable 'BuildNumber' with buildId
+            context.SetVariable("BuildNumber", pipelineId.ToString());
```

Comment:
Should be PipelineId to avoid confusion.

Reply:
Naming it "BuildNumber" so it's consistent with build artifacts tasks.

Reply:
I would strongly recommend renaming it to avoid confusion. Especially considering the fact that "Download build artifacts" never made it past version 0 (which tells me it has never made it past preview).